Tests/RxCocoaTests/Signal+Test.swift

```diff
@@ -527,3 +527,123 @@ extension SignalTests {
         XCTAssertEqual(latest, 1)
     }
 }
+
+// MARK: Emit to replay relay
+
+extension SignalTests {
+    func testEmitReplayRelay() {
+        let relay = ReplayRelay<Int>.create(bufferSize: 1)
+
+        var latest: Int?
+        _ = relay.subscribe(onNext: { latestElement in
+            latest = latestElement
+        })
+
+        _ = (Signal.just(1) as Signal<Int>).emit(to: relay)
+
+        XCTAssertEqual(latest, 1)
+    }
+
+    func testEmitReplayRelays() {
+        let relay1 = ReplayRelay<Int>.create(bufferSize: 1)
+        let relay2 = ReplayRelay<Int>.create(bufferSize: 1)
+
+        var latest1: Int?
+        var latest2: Int?
+
+        _ = relay1.subscribe(onNext: { latestElement in
+            latest1 = latestElement
+        })
+
+        _ = relay2.subscribe(onNext: { latestElement in
+            latest2 = latestElement
+        })
+
+        _ = (Signal.just(1) as Signal<Int>).emit(to: relay1, relay2)
+
+        XCTAssertEqual(latest1, 1)
+        XCTAssertEqual(latest2, 1)
+    }
+
+    func testEmitOptionalReplayRelay1() {
+        let relay = ReplayRelay<Int?>.create(bufferSize: 1)
```

Comment:
not seeing any tests for big buffer sizes - without these, this is basically identical to Behavior Relay.

Please make tests for some random amount of elements showcasing having less than max, more than max, zero elements, etc. This seems to me like the essence worth testing of this relay.

Reply:
I added the tests I could think of.

I noticed there are not tests for ReplaySubject with bigger buffer size, so I added some there as well.
Considering ReplayRelay is a wrapper around ReplaySubject, the tests I added in `ReplayRelayTests.swift` seem a bit redundant. 

Let me know what you think and if you have any cases I didn't think of.